images/ubuntu/static.yaml

```diff
@@ -0,0 +1,29 @@
+# Builds a "distroless" image based on ubuntu:jammy
+#
+# "static" is a image that can be used to package statically built binaries (go, rust, etc)
+
+build:
+  from:
+    type: docker
+    url: docker://ubuntu:jammy
+  run: |
+    apt-get update && apt-get dist-upgrade -y
```

Comment:
i think dist-upgrade is probably not necessary here, but not harmful other than IO

Reply:
just to get the very latest, if that!

Reply:
the 'apt-get update' will ensure that what you download will be the latest.  I wasn't sure what the behavior of '--reinstall' would be, because it is quite possible that apt *can't* re-download the same version it had.  So I tested.  Below I had an out-of-date base-files and did a 'reinstall' and it got the new version.  

So you can do the 'dist-upgrade` if you want, but it will not affect the downloaded files only the OS that is doing the downloading.


```
root@f1:~# apt-get update
Hit:1 http://archive.ubuntu.com/ubuntu focal InRelease
...
Fetched 7480 kB in 13s (555 kB/s)
Reading package lists... Done

root@f1:~# dpkg-query --show base-files
base-files      11ubuntu5.5

root@f1:~# apt-cache policy base-files
base-files:
  Installed: 11ubuntu5.5
  Candidate: 11ubuntu5.6
  Version table:
     11ubuntu5.6 500
        500 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages
 *** 11ubuntu5.5 100
        100 /var/lib/dpkg/status
     11ubuntu5 500
        500 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages

root@f1:~# apt-get install --reinstall base-files
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following package was automatically installed and is no longer required:
  libfreetype6
Use 'apt autoremove' to remove it.
The following packages will be upgraded:
  base-files
1 upgraded, 0 newly installed, 0 to remove and 6 not upgraded.
Need to get 60.4 kB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 base-files amd64 11ubuntu5.6 [60.4 kB]
Fetched 60.4 kB in 0s (1725 kB/s)
(Reading database ... 32012 files and directories currently installed.)
Preparing to unpack .../base-files_11ubuntu5.6_amd64.deb ...
Warning: Stopping motd-news.service, but it can still be activated by:
  motd-news.timer
Unpacking base-files (11ubuntu5.6) over (11ubuntu5.5) ...

```